src/libraries/System.IO.Pipes/tests/NamedPipeTests/NamedPipeTest.CurrentUserOnly.Windows.cs

```diff
@@ -132,8 +133,8 @@ public NamedPipeTest_CurrentUserOnly_Windows(TestAccountImpersonator testAccount
             _testAccountImpersonator = testAccountImpersonator;
         }
 
-        [OuterLoop]
-        [ConditionalFact(nameof(IsAdminOnSupportedWindowsVersions))]
+        [OuterLoop("Requires admin privileges")]
```

Comment:
Is this the reason it's OuterLoop?  That's acovered by the ConditionalFact condition, and there are other tests that require admin privileges that aren't OuterLoop.